### PR TITLE
Migration to new Organization / Plugin Layout

### PR DESCRIPTION
**Pull request by wombelix**
Hey @justinmayer,

thanks for your assistance, as discussed in https://github.com/getpelican/pelican-plugins/issues/1328, I added my changes to a fresh fork of the new https://github.com/pelican-plugins/pdf Repository.

Please let me know if you are happy with the Changes or if there is anything else i should adjust prior this initial PR can be merged.

Cheers
Dominik

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  PYTEST_ADDOPTS: "--color=yes"
+
+jobs:
+  test:
```
Comment (justinmayer) [inline .github/workflows/main.yml:10]: Would you do me a favor and add a newline between `jobs:` and `test:`? It adds a bit of visual separation that I think aids readability.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      - name: Install dependencies
+        run: |
+          poetry run pip install --upgrade pip
+          poetry install
+      - name: Run linters
+        run: poetry run invoke lint
+
+
+  deploy:
+    name: Deploy
```
Comment (justinmayer) [inline .github/workflows/main.yml:76]: [I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/04efda54a9ad4a904869a8b64f70d4c312c46d34) Right below this line, please add:
```
environment: Deployment
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Check release
+        id: check_release
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install poetry githubrelease autopub
```
Comment (justinmayer) [inline .github/workflows/main.yml]: [I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/93509377a60082b846c41e21932b5f68a40bd3e0) Please add `httpx==0.16.1` to this list of packages.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+PDF Generator: A Plugin for Pelican
+====================================================
```
Comment (justinmayer) [inline README.md]: Could you shorten the separator so it matches the length of the line above?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+This plugin can be installed via:
+
+    python -m pip install pelican-pdf
+
+Usage
+-----
+
+To customize the PDF output, you can use the following settings in your
+Pelican configuration file::
```
Comment (justinmayer) [inline README.md]: We can probably remove (1) the hard line wrap and (2) the extra colon on the end of the sentence.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+	PDF_STYLE = ''
+	PDF_STYLE_PATH = ''
+
+`PDF_STYLE_PATH` defines a new path where *rst2pdf* will look for style sheets, while ``PDF_STYLE`` specifies the style you want to use.
```
Comment (justinmayer) [inline README.md]: ``PDF_STYLE`` ⇢ `PDF_STYLE`

File: README.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+For a description of the available styles, please read the [rst2pdf documentation](http://rst2pdf.ralsina.me/handbook.html#styles).
+
+Known Issues
+------------
+
+Relative links in the form of `|filename|images/test.png` are not yet handledby the PDF generator.
```
Comment (justinmayer) [inline README.md]: handledby ⇢ handled by

File: README.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+Contributors
+------------
+
+Primary Author and earlier Contributor
+
+- Kyle Mahan, Renato Cunha, dpetzel, Justin Mayer, Lucas Cimon
```
Comment (justinmayer) [inline README.md]: Perhaps change this section to read:

> Contributors include: Kyle Mahan, Renato Cunha, dpetzel, Justin Mayer, and Lucas Cimon

File: pyproject.toml [hunk 9]
```diff
@@ -0,0 +1,76 @@
+[tool.poetry]
+name = "pelican-pdf"
+version = "0.0.0"
+description = "The PDF Generator plugin automatically exports articles and pages as PDF files as part of the site generation process"
+authors = ["Pelican Dev Team <authors@getpelican.com>"]
+license = "AGPL-3.0"
+readme = "README.md"
+keywords = ['pelican', 'plugin', 'pdf', 'generator']
```
Comment (justinmayer) [inline pyproject.toml]: Perhaps use double-quotation marks here for consistency?

File: pyproject.toml [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+pytest-sugar = "^0.9.4"
+Werkzeug = "^1.0"
+
+[tool.poetry.extras]
+markdown = ["markdown"]
+
+[tool.autopub]
+project-name = "PDF Generator"
+git-username = "botpub"
+git-email = "botpub@autopub.rocks"
```
Comment (justinmayer) [inline pyproject.toml:57]: [I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/93509377a60082b846c41e21932b5f68a40bd3e0) Please add the following line to this section:
```toml
append-github-contributor = true
```

File: pyproject.toml [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+[tool.isort]
+# Maintain compatibility with Black
+combine_as_imports = true
+force_grid_wrap = 0
+include_trailing_comma = true
+line_length = 88
```
Comment (justinmayer) [inline pyproject.toml]: [I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/05fdcbef84a62ce7015b5eb2673edee9637f6fcd) Please replace the above four lines with:
```toml
profile = "black"
```

File: tox.ini [hunk 12]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
+ignore = E203, E266, E501, W503
```
Comment (justinmayer) [inline tox.ini]: [I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/b83e86cc39f947ab3ee57d3d948e54298f54ae6a) Please change the ignore list to:
```
ignore = E203, W503
```

**Review by justinmayer — CHANGES_REQUESTED** [review]
Well done, Dominik! I make a few minor change suggestions, many of which are my fault because I forgot to push some local changes to the CookieCutter template. Mea culpa! 😅

Many thanks for helping out with this migration. Please let me know if there is anything I can clarify!

File: README.md (hunk 5 above)
Comment (wombelix) [reply README.md]: Indeed, Copy & Paste issue from the previous RST based Readme

**Comment by wombelix** [comment]
Thank you very much Justin, everything changed as suggested :)

**Review by justinmayer — APPROVED** [review]
Looks great. Many thanks for making those changes!

**Comment by justinmayer** [comment]
Since I melded your fixes into the relevant commits, I have rebased and manually merged the four commits to `main`. Thanks again for all your work on this! 🏅

**Pull request closed without merging**